core/templates/css/oppia.css

```diff
@@ -4516,3 +4524,224 @@ div.oppia-issues-learner-action {
+    padding: 55px 70px 30px 0;
+    text-align: left;
+    width: 100%;
+  }
+
+  .oppia-welcome-guide-exploration-image {
+    padding: 30px 0px 30px 70px;
+    width: 100%;
+    text-align: center;
+  }
+}
+
+@media (max-width: 992px) {
+  .oppia-welcome-guide-img {
+    text-align: center;
+    padding: 20px 0 30px 0;
+    height: auto;
+  }
+
+  .oppia-welcome-guide-header {
+    padding: 70px 70px 0 70px;
+    height: 300px;
+  }
+
+  .oppia-welcome-guide-img img {
+    width: 80%;
+    height: auto;
+    padding-top: 0px;
+  }
+
+  .oppia-welcome-guide-fractions-image img {
+    width: 90%;
+  }
+
+  .oppia-welcome-guide-exploration-image {
+    padding: 30px 15% 40px 15%;
+    text-align: center;
+    width: 100%;
+  }
+
+  .oppia-welcome-guide-browser-image {
+    padding: 0 15% 30px 15%;
+  }
+}
+
+@media (max-width: 700px) {
+  .oppia-welcome-guide-img {
+    height: auto;
+    padding-bottom: 30px;
+    text-align: center;
+  }
+
+  .oppia-welcome-guide-header {
+    height: 400px;
+    padding: 40px 70px 0 70px;
+  }
+
+  .oppia-welcome-guide-img img {
+    width: auto;
+    height: auto;
+    padding: 15px 50px 0 50px;
+  }
+
+  .oppia-welcome-guide-second-section {
+    padding: 0 70px 0 70px;
+  }
+}
+
+.oppia-welcome-guide-numbers {
+  display: inline;
+  font-family: "Roboto", Arial, sans-serif;
+  font-size: 1.17em;
+  padding-top: 0px;
+  padding-bottom: 0px;
+  text-align: left;
+}
+
+.oppia-welcome-guide-h1 {
+  color: #005c5e;
+  font-family: "Capriola", "Roboto", Arial, sans-serif;
+  padding-bottom: 15px;
+}
+
+.oppia-welcome-guide-p {
+  font-family: "Roboto", Arial, sans-serif;
+  font-size: 1.17em;
+  text-align: left;
+  line-height: 200%;
+}
+
+span.oppia-welcome-guide-blue {
+  background: rgb(13,61,245);
+  border-radius: 1em;
+  -moz-border-radius: 1em;
+  -webkit-border-radius: 1em;
+  color: #ffffff;
+  display: inline-block;
+  font-size: 1.5em;
+  font-weight: bold;
+  line-height: 2em;
+  margin-right: 15px;
+  text-align: center;
+  width: 2em;
+}
+
+span.oppia-welcome-guide-purple {
+  background: rgb(116,39,245);
+  border-radius: 1em;
+  -moz-border-radius: 1em;
+  -webkit-border-radius: 1em;
+  color: #ffffff;
+  display: inline-block;
+  font-size: 1.5em;
+  font-weight: bold;
+  line-height: 2em;
+  margin-right: 15px;
+  text-align: center;
+  width: 2em;
+
+}
+
+span.oppia-welcome-guide-red {
+  background: rgb(231,67,57);
+  border-radius: 1em;
+  -moz-border-radius: 1em;
+  -webkit-border-radius: 1em;
+  color: #ffffff;
+  display: inline-block;
+  font-size: 1.5em;
+  font-weight: bold;
+  line-height: 2em;
+  margin-right: 15px;
+  text-align: center;
+  width: 2em;
+}
+
+span.oppia-welcome-guide-orange {
+  background: rgb(236,147,69);
+  border-radius: 1em;
+  -moz-border-radius: 1em;
+  -webkit-border-radius: 1em;
+  color: #ffffff;
+  display: inline-block;
+  font-size: 1.5em;
+  font-weight: bold;
+  line-height: 2em;
+  margin-right: 15px;
+  text-align: center;
+  width: 2em;
+
```

Comment:
mm, why this empty line here? Remove this.

Reply:
fixed